user-guide/Advanced_Functionality/SNMP_managers/DataMiner_SNMP_notification_definition/Custom_DataMiner_notification.md

```diff
@@ -48,7 +48,7 @@ Syntax:
 | User status ID | 1.3.6.1.4.1.8813.1.1.2.4.1.10| Possible values:<br> - Not assigned (18)<br> - Acknowledged (19)<br> - Resolved (20)<br> - Unresolved (21) |
 | Alarm root ID | 1.3.6.1.4.1.8813.1.1.2.4.1.11| |
 | Source ID | 1.3.6.1.4.1.8813.1.1.2.4.1.12| Possible values:<br> - DataMiner (16)<br> - Correlation engine (23)<br> - Automation engine (26) |
-| Table display index| 1.3.6.1.4.1.8813.1.1.2.4.1.13| |
+| Table display index| 1.3.6.1.4.1.8813.1.1.2.4.1.13| What is returned depends on how the table is indexed:<br>- Primary key only: Returns primary key value.<br>- Table with naming option, but no display key column: Returns what has been defined with the naming option.<br>- Table with naming option and column of type DisplayKey: Returns the value in the display key column.|
```

Comment:
No mention of Display key only?

Reply:
As this will need to be tested & verified, and this will take some time, I will already merge this PR, because the added info is already an improvement. We can then add the missing info later.